### PR TITLE
added 'affinity' and 'tolerations' configuration hooks to cert-manager

### LINKED ISSUE #7042: cert-manager does not have affinity and tolerations configuration
At the moment there is no way to specify affinity and tolerations for the cert-manager deployment.

### PR DESCRIPTION
**Pull request by gintautassulskus**
Added affinity and tolerations configuration options to cert-manager chart to fix: https://github.com/helm/charts/issues/7042

### CODE REVIEW

**Comment by gintautassulskus** [comment]
Do we need lgtm label for this?

File: stable/cert-manager/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: cert-manager
-version: v0.4.0
+version: v0.4.1
```
Comment (cpanato) [inline stable/cert-manager/Chart.yaml]: Please bump to 1.0.0 . We had a call yesterday in the charts meeting we the approach will be all stable charts which is in  version 0.x.x should be at least 1.0.0

Cc @davidkarlsen @mattfarina

**Comment by cpanato** [comment]
Also you will need to rebase and fix the conflict

**Comment by munnerz** [comment]
Hi there -

Thanks for your pull request. I'm afraid we don't accept changes to the cert-manager Helm chart in this repository, in favour of making the changes in the upstream [jetstack/cert-manager](https://github.com/jetstack/cert-manager) repo.

If you wouldn't mind making this same change there, it will then be synced across automatically. This allows us to fully test our Helm chart/deployment code before it is published to users 😄 

You can find a copy of the chart here: https://github.com/jetstack/cert-manager/tree/master/contrib/charts/cert-manager

Again, thanks very much for doing this work and I'd love to be able to accept this upstream 😄 

/close

**Comment by munnerz** [comment]
... seems like someone has disabled the close plugin for this repo :\ can this be re-enabled so I am able to manage PRs?

/cc @mattfarina

File: stable/cert-manager/Chart.yaml (hunk 1 above)
Comment (munnerz) [reply stable/cert-manager/Chart.yaml]: If that's the case - we'll need to move cert-manager out of the 'stable' repository, as we are not currently committed to the API surface exposed in values.yaml between minor versions.

This is probably a conversation better had elsewhere though 😄

**Comment by gintautassulskus** [comment]
Ok, here it is: https://github.com/jetstack/cert-manager/pull/869

**Pull request closed without merging**